src/app/personside/infotabs/utbetalinger/totalt utbetalt/TotaltUtbetalt.tsx

```diff
@@ -78,24 +110,34 @@ class TotaltUtbetalt extends React.Component<TotaltUtbetaltProps, State> {
             [[periode, brutto, trekk, utbetalt]]);
 
         return (
-            <Wrapper onClick={() => cancelIfHighlighting(this.toggleVisDetaljer)}>
-                <Header>
-                    <Undertittel>Totalt utbetalt for perioden</Undertittel>
-                    <TotaltUtbetaltOversikt>
-                        <Normaltekst tag="span">
-                            {totaltUtbetaltTabell}
-                        </Normaltekst>
-                    </TotaltUtbetaltOversikt>
-                    <FlexEnd>
-                        <PrintKnapp onClick={() => console.log('ikke implementert')}/>
-                    </FlexEnd>
-                </Header>
-                <TotaltUtbetaltDetaljer
-                    visDetaljer={this.state.visDetaljer}
-                    toggleVisDetaljer={this.toggleVisDetaljer}
-                    {...this.props}
-                />
-            </Wrapper>
+            <Printer getPrintTrigger={(trigger: () => void) => this.print = trigger}>
```

Comment:
trenger du typedeklarasjonen her..? kasnkje bare `{trigger => this.print = trigger}`?